helm/idol-answerserver/templates/new_config.yaml

```diff
@@ -0,0 +1,24 @@
+# BEGIN COPYRIGHT NOTICE
+# Copyright 2024 Open Text.
+# 
+# The only warranties for products and services of Open Text and its affiliates and licensors
+# ("Open Text") are as may be set forth in the express warranty statements accompanying such
+# products and services. Nothing herein should be construed as constituting an additional warranty.
+# Open Text shall not be liable for technical or editorial errors or omissions contained herein.
+# The information contained herein is subject to change without notice.
+#
+# END COPYRIGHT NOTICE
+
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: idol-answerserver-cfg
+  labels: {{- include "idol-library.labels" (dict "root" . "component" .Values ) | nindent 4 }}
+
+data:
+  answerserver.cfg: |
+{{ tpl (print "resources/answerserver.cfg" | .Files.Get) . | indent 4 }}
+
+  config.sh: |
+{{ tpl (print "resources/config.sh" | .Files.Get) . | indent 4 }}
```

Comment:
This should just include config.sh, not answerserver.cfg. File probably wants to be renamed (perhaps configmap-prestart.yaml). This will then need to be in additionalVolumes/additionalVolumeMounts